graphql/resolvers/util/ui_plugin_build_baron.go

```diff
@@ -1,12 +1,12 @@
-package graphql
+package util
```

Comment:
I don't think we need this package. By defining the util package we collide with the existing `util` package. 
This function is only used by Graphql we don't need to place it in a separate package and expose it as a public function.